public/app/features/api-keys/ApiKeysPage.tsx

```diff
@@ -51,12 +51,15 @@ interface OwnProps {}
 export type Props = OwnProps & ConnectedProps<typeof connector>;
 
 interface State {
-  isAdding: boolean;
+  migrationSummaryVisible: boolean;
```

Comment:
maybe it's a better name?
```suggestion
  showMigrationResult: boolean;
```

Reply:
agree